### PR TITLE
Fix for missing appIds

### PR DESCRIPTION
**Pull request by wpride**
Some migrated incomplete forms were missing appId's which caused errors as in http://manage.dimagi.com/default.asp?245942

This migration addresses that by using either the linked menu session or the postUrl depending on what is available. This migration is very janky and I am not proud of that. However its also very resourceful and I am proud of that 😑

### CODE REVIEW

File: src/main/java/db/migration/V11__form_session_add_missing_appid.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ * Attempt to fix null appId in form session
+ */
+public class V11__form_session_add_missing_appid implements SpringJdbcMigration {
+    @Override
+    public void migrate(final JdbcTemplate jdbcTemplate) throws Exception {
+        String sql = "SELECT * FROM formplayer_sessions WHERE COALESCE(appid, '') = ''";
```
Comment (benrudolph) [inline src/main/java/db/migration/V11__form_session_add_missing_appid.java]: `appid is NULL` is the idiomatic way to get null values

Comment (wpride) [inline src/main/java/db/migration/V11__form_session_add_missing_appid.java]: That wasn't working for me actually

Comment (wpride) [inline src/main/java/db/migration/V11__form_session_add_missing_appid.java]: http://stackoverflow.com/questions/14172978/postgresql-select-empty-fields

Comment (benrudolph) [inline src/main/java/db/migration/V11__form_session_add_missing_appid.java]: well i'll be damned

**Pull request merged**